site/docs/reference/Connectors/capture-connectors/hubspot.md

```diff
@@ -0,0 +1,116 @@
+* [Engagements Notes](https://developers.hubspot.com/docs/api/crm/notes)
+* [Engagements Tasks](https://developers.hubspot.com/docs/api/crm/tasks)
+* [Feedback Submissions](https://developers.hubspot.com/docs/api/crm/feedback-submissions)
+* [Forms](https://developers.hubspot.com/docs/api/marketing/forms)
+* [Form Submissions](https://legacydocs.hubspot.com/docs/methods/forms/get-submissions-for-a-form)
+* [Line Items](https://developers.hubspot.com/docs/api/crm/line-items)
+* [Marketing Emails](https://legacydocs.hubspot.com/docs/methods/cms_email/get-all-marketing-email-statistics)
+* [Owners](https://developers.hubspot.com/docs/methods/owners/get_owners)
+* [Products](https://developers.hubspot.com/docs/api/crm/products)
+* [Property History](https://legacydocs.hubspot.com/docs/methods/contacts/get_contacts)
+* [Quotes](https://developers.hubspot.com/docs/api/crm/quotes)
+* [Subscription Changes](https://developers.hubspot.com/docs/methods/email/get_subscriptions_timeline)
+* [Tickets](https://developers.hubspot.com/docs/api/crm/tickets)
+* [Ticket Pipelines](https://developers.hubspot.com/docs/api/crm/pipelines)
+* [Workflows](https://legacydocs.hubspot.com/docs/methods/workflows/v3/get_workflows)
+
+By default, each resource associated with your Hubspot account is mapped to a Flow collection through a separate binding.
+
+## Prerequisites
+
+There are two ways to authenticate with Hubspot when capturing data: using OAuth, and with an API key.
+Their prerequisites differ.
+
+OAuth is recommended for simplicity in the Flow web app;
+the API key method is the only supported method using the command line.
+
+### Prerequisites for OAuth
+
+:::caution Beta
+OAuth implementation is under active development and is coming soon.
+Use the API key method for now.
+:::
+
+* A Hubspot account
+
+### Prerequisites using an API key
+
+* A Hubspot account
+
+* A Hubspot [API key](https://knowledge.hubspot.com/integrations/how-do-i-get-my-hubspot-api-key)
+
+## Configuration
+
+You configure connectors either in the Flow web app, or by directly editing the catalog spec YAML.
+See [connectors](../../../concepts/connectors.md#using-connectors) to learn more about using connectors. The values and YAML sample below provide configuration details specific to the Hubspot source connector.
+
+### Properties
+
+#### Endpoint
+
+The following properties reflect the API Key authentication method.
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| **`/credentials`** | Authentication mechanism | Choose how to authenticate to HubSpot. | object | Required |
+| **`/credentials/credentials_title`** | Credentials set | Type of credentials. Set to `API Key Credentials` | string | Required |
+| **`/credentials/api_key`** | API Key | Your Hubspot API key | string | Required |
+| **`/start_date`** | Start Date | UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. | string | Required |
```

Comment:
I introduced a new option yesterday: https://github.com/estuary/airbyte/runs/6403948874?check_suite_focus=true#logs

Basically, you get access to Feedback Submissions, Marketing Emails and Workflows streams only if `subscription_type: pro`, otherwise those streams are not enabled, because only pro users have access to those.